### PR TITLE
Swmaas 679

### PR DESCRIPTION
**Pull request by dnguyen-phunware**

### CODE REVIEW

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt [hunk 1]
```diff
@@ -205,7 +202,26 @@ class VoicePromptActivity : AppCompatActivity(), OnPhunwareMapReadyCallback,
...
+            }
+
+            if(turnManeuver != null && position < navigator.maneuvers.size - 1 && turnable)
+            {
+                text += " then "
+                text += displayHelper.stringForDirection(this, turnManeuver)
+            }
+            else
+            {
+                text += " to arrive at the destination."
```
Comment (avakkachen) [inline Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt]: Can we move these strings to strings.xml

File: Samples/MapScenarios/VoicePrompt.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+## Sample - Voice Prompt
```
Comment (avakkachen) [inline Samples/MapScenarios/VoicePrompt.md]: Voice prompt Read me looks good

File: Samples/MapScenarios/OffRouteAlerts.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+## Sample - Off Route
```
Comment (avakkachen) [inline Samples/MapScenarios/OffRouteAlerts.md]: Read me looks good

File: Samples/MapScenarios/VoicePrompt.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        //if voice is enabled 
+        if (voiceEnabled) {
+            val pair = navOverlay.getManeuverPair()
+            var text = displayHelper.stringForDirection(this, pair.mainManeuver)
+
+            val turnManeuver  = pair.turnManeuver
+            var turnable : Boolean  = false
+            if(turnManeuver != null)
```
Comment (avakkachen) [inline Samples/MapScenarios/VoicePrompt.md]: Let's do Java/Kotlin style code formatting for code snippets too.

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt [hunk 5]
```diff
@@ -177,4 +177,9 @@ class NavigationOverlayView @JvmOverloads constructor(context: Context, attrs: A
 
         override fun isViewFromObject(view: View, `object`: Any): Boolean = view == `object`
     }
+
+    fun getManeuverPair() : ManeuverPair
+    {
+        return adapter.getItem(currentItem)
+    }
```
[the remark this answers is not in the document]
Comment (dnguyen-phunware) [reply Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt]: done

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt (hunk 1 above)
Comment (dnguyen-phunware) [reply Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt]: done

File: Samples/MapScenarios/VoicePrompt.md (hunk 4 above)
Comment (dnguyen-phunware) [reply Samples/MapScenarios/VoicePrompt.md]: will do this in the other PR

File: Samples/MapScenarios/OffRouteAlerts.md (hunk 3 above)
Comment (dnguyen-phunware) [reply Samples/MapScenarios/OffRouteAlerts.md]: other PR

File: Samples/MapScenarios/VoicePrompt.md (hunk 4 above)
Comment (dnguyen-phunware) [reply Samples/MapScenarios/VoicePrompt.md]: done

**Pull request merged**